src/algo/depth-to-rgb-calibration/valid-scene.cpp

```diff
@@ -702,13 +702,14 @@ bool optimizer::is_scene_valid()
 
     // remove pixels in section map that were removed in weights
     AC_LOG(DEBUG, "... " << _z.supressed_edges.size() << " total edges");
-    for (auto i = 0; i < _z.supressed_edges.size(); i++)
+    /*for (auto i = 0; i < _z.supressed_edges.size(); i++)
```

Comment:
Remove?

Reply:
it is related to scene validity, I will keep it for now